_includes/footer.html

```diff
@@ -89,7 +81,6 @@
                     <li class="fa fa-github"><a href="https://github.com/docker">GitHub</a></li>
                     <li class="fa fa-linkedin"><a href="https://www.linkedin.com/company/docker">Linkedin</a></li>
                     <li class="fa fa-facebook"><a href="https://www.facebook.com/docker.run">Facebook</a></li>
-                    <li class="fa fa-reddit"><a href="http://www.reddit.com/r/docker">Reddit</a></li>
```

Comment:
We don't actively use Reddit? (note: if we bring it back, we should use `https://` instead of `http://`